pytorch_pfn_extras/training/manager.py

```diff
@@ -505,25 +550,30 @@ def set_training_started(engine):
             # Make all the next
             # handlers to be executed after user defined ones
             @self.engine.on(Events.ITERATION_COMPLETED)
-            def run_extensions_on_iter(engine):
+            def run_extensions_on_iter(engine: Engine):
                 self.iteration = engine.state.iteration
                 self.run_extensions()
 
             # This should be the last extension to be run
             @self.engine.on(Events.ITERATION_COMPLETED)
-            def close_reporter_on_iter(engine):
+            def close_reporter_on_iter(engine: Engine):
                 self.cm.__exit__(None, None, None)
 
         @self.engine.on(Events.COMPLETED)
-        def set_extensions_cleanup(engine):
+        def set_extensions_cleanup(engine: Engine):
             self._finalize_extensions()
 
-    def state_dict(self):
+    def state_dict(self, *, transform_models=None) -> Dict[str, Any]:
```

Comment:
`BaseExtensionsManager.state_dict` supports `transform_models` option, but this class did not.

Reply:
Done in 314c2af.